src/Catalog/Schema.cs

```diff
@@ -153,6 +153,13 @@ public static class Predicates
             public static readonly Uri LicenseFile = new Uri(Prefixes.NuGet + "licenseFile");
 
             public static readonly Uri IconFile = new Uri(Prefixes.NuGet + "iconFile");
+
+            public static readonly Uri Deprecation = new Uri(Prefixes.NuGet + "deprecation");
+            public static readonly Uri DeprecationReasons = new Uri(Prefixes.NuGet + "reasons");
```

Comment:
`deprecationReasons`

Reply:
I think all 5 of these latter deprecation predicates should include "deprecation" in their URI

Reply:
I tried that, but what ended up happening was the object had the shape
```
"deprecation": {
    "deprecationReasons": [ ... ],
    "deprecationMessage": "...",
    "..."
}
```
which I didn't want.

If you know a way to change this URI in a way that makes it unique and keeps the name, I'd love too.

Reply:
does `deprecation/reasons` work?

Reply:
I don't know all of JSON-LD conventions.

Reply:
I think I tried it but I will double check. If I remember correct it did some very very strange things.

Reply:
Either the field name is made more generic or the URI is made more specific. the current name of the field is very specific but the URI is very generic

Reply:
Notice all other predicates today have the field name exactly matching the URI suffix. I would go that route, personally by removing "deprecation" from the field name

Reply:
`#deprecation/reasons` becomes `http://schema.nuget.org/reasons` in the output...I will try to find a better way

Reply:
Figured out a way to do it using `@context`...but it's very ugly and will break if we add any other properties with the same name. I will make the field name more generic instead.

In case you were curious...

In `Schema.cs`:
```
            public static readonly Uri DeprecationReasons = new Uri(Prefixes.NuGet + "deprecationReasons");
            public static readonly Uri DeprecationMessage = new Uri(Prefixes.NuGet + "deprecationMessage");
            public static readonly Uri DeprecationAlternatePackage = new Uri(Prefixes.NuGet + "deprecationAlternatePackage");
            public static readonly Uri DeprecationAlternatePackageId = new Uri(Prefixes.NuGet + "deprecationAlternatePackageId");
            public static readonly Uri DeprecationAlternatePackageRange = new Uri(Prefixes.NuGet + "deprecationAlternatePackageRange");
```

In `PackageDetails.json`:
```
        "reasons" : { "@id": "deprecationReasons", "@container" : "@set" },
        "message" : { "@id": "deprecationMessage" },
        "alternatePackage" : { "@id": "deprecationAlternatePackage" },
        "id" : { "@id": "deprecationAlternatePackageId" },
        "range" : { "@id": "deprecationAlternatePackageRange" }
```